### ISSUE TITLE
doesnt work for me?

### ISSUE THREAD

**Issue by EnteSuesssauer15** [LinuxWillWin1/Assetto-Corsa-on-SteamDeck#3]
it just wont do anything it wont crash or open it just sits there for ours on end starting but not running

**Comment by notRobertBarnes** [comment]
Having the same issue, did you ever manage to resolve it?

**Comment by EnteSuesssauer15** [comment]
Nope, I eventually gave up. Literally every paper I could find about on the internet didn't work for me.
